client/lib/analytics/index.js

```diff
@@ -19,6 +19,8 @@ var config = require( 'config' ),
 	_superProps,
 	_user;
 
+import { retarget } from 'lib/analytics/ad-tracking';
```

Comment:
looks like the failing test is using file system mocks to mock out config and the mock doesn't have `isEnabled` defined.